IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m

```diff
@@ -776,33 +980,75 @@ - (BOOL)saveJsonObject:(MSIDJsonObject *)jsonObject
 
 #pragma mark - Account metadata
 
-- (MSIDAccountMetadataCacheItem *)accountMetadataWithKey:(MSIDAccountMetadataCacheKey *)key serializer:(id<MSIDExtendedCacheItemSerializing>)serializer context:(id<MSIDRequestContext>)context error:(NSError *__autoreleasing *)error {
-    [self createUnimplementedError:error context:context];
-    return nil;
+- (MSIDAccountMetadataCacheItem *)accountMetadataWithKey:(MSIDAccountMetadataCacheKey *)key
```

Comment:
nit: add a TODO to improve the saving logic here (to not pollute keychain)

Reply:
Added.